File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/responses/CancelOrderResponse.java
package com.elgin.elginexperience.shipay.responses;

import com.google.gson.annotations.SerializedName;

public class CancelOrderResponse {
    @SerializedName("order_id")
    public String order_id;

    @SerializedName("status")
    public String status;
}

File: Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/responses/CancelOrderResponse.kt
package com.example.e1_elgin_kotlin.shipay.responses

import com.google.gson.annotations.SerializedName

data class CancelOrderResponse (
    @SerializedName("order_id")
    var order_id: String,

    @SerializedName("status")
    var status: String
)
